packages/wrangler/src/pages/publish.tsx

```diff
@@ -348,6 +346,7 @@ export const Handler = async ({
 			}
 
 			_routes = JSON.stringify(optimizeRoutesJSONSpec(advancedModeRoutes));
+			formData.append("_routes.json", new File([_routes], "_routes.json"));
```

Comment:
Isn't `_routes` undefined here?

Reply:
No, it's defined a ways up and then set right above this on line 348.
I tested this to be sure

Reply:
oh wow I'm a dumb dumb totally missed line 348 😅